packages/ts/file-router/src/runtime/createMenuItems.ts

```diff
@@ -30,7 +30,13 @@ export function createMenuItems(vaadinObject = (window as VaadinWindow).Vaadin):
             to: _path,
             icon: config.menu?.icon,
             title: config.menu?.title ?? config.title,
+            order: config.menu?.order,
           };
         })
+        // Sort views according to the order specified in the view configuration.
+        .sort((menuA, menuB) => {
+          const ordersDiff = (menuA.order ?? Number.MAX_VALUE) - (menuB.order ?? Number.MAX_VALUE);
+          return ordersDiff !== 0 ? ordersDiff : menuA.to.localeCompare(menuB.to);
```

Comment:
I guess, you have to use `Intl.Collator` here as it is the most correct way to do the locale comparison nowdays.